ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -968,9 +966,11 @@ const triggerSet: TriggerSet<Data> = {
       // DC6 Critical Underflow Bug (red)
       // Debuffs last 27s
       netRegex: { effectId: ['D65', 'DC6'] },
+      // TODO: should we have a "Watch Rot" call if you don't get it?
+      // (with some suppression due to inconsistent rot pickup timings etc)
       condition: Conditions.targetIsYou(),
       delaySeconds: (_data, matches) => parseFloat(matches.duration) - 3,
-      alertText: (_data, _matches, output) => output.spread!(),
+      infoText: (_data, _matches, output) => output.spread!(),
```

Comment:
This is just my own personal preference (feel free to disagree) but alert felt too much severity here.  It could just be our strat, where rot people plant and other people move away from them.

Reply:
That's a good thought.  I've adjusted those severities.  I also changed the wording on the defamation color to "red *is* defamation" rather than just "red defamation" (which sounds like maybe the player is red defamation, now that you've added a bunch more calls).